src/components/ProductCard/styles.module.scss

```diff
@@ -0,0 +1,120 @@
+@import '../../assets/styles/variables';
+@import '../../assets/styles/reset';
```

Comment:
You don't need this import here, it is enough to add it to main.scss. This way you only need to import variables, mixins, functions, and so on if you use them in this file

Reply:
```suggestion
;
```